engine/orchestration/src/main/java/com/cloud/agent/manager/AgentManagerImpl.java

```diff
@@ -1759,8 +1764,11 @@ public void processConnect(final Host host, final StartupCommand cmd, final bool
             if (cmd instanceof StartupRoutingCommand) {
                 if (((StartupRoutingCommand)cmd).getHypervisorType() == HypervisorType.KVM || ((StartupRoutingCommand)cmd).getHypervisorType() == HypervisorType.LXC) {
                     Map<String, String> params = new HashMap<String, String>();
-                    params.put(Config.RouterAggregationCommandEachTimeout.toString(), _configDao.getValue(Config.RouterAggregationCommandEachTimeout.toString()));
                     params.put(Config.MigrateWait.toString(), _configDao.getValue(Config.MigrateWait.toString()));
+                    params.put(ROUTER_AGGREGATION_COMMAND_EACH_TIMEOUT, _configDao.getValue(ROUTER_AGGREGATION_COMMAND_EACH_TIMEOUT));
+                    params.put(KVM_VM_MIGRATE_SPEED_STRING, _configDao.getValue(KVM_VM_MIGRATE_SPEED_STRING));
```

Comment:
KVM_VM_MIGRATE_SPEED is a cluster-level setting, but global config is uses here.